File: projects/m1/013-birth-date-to-astrological-sign/solutions/jimmyhu/solution-birth-date-to-astrological-sign.py
months = ['january','february','march','april','may','june','july','august','september','october','november','december']
sign = ['Capricorn','Aquarius','Pisces','Aries','Taurus','Gemini','Cancer','Leo','Virgo','Libra','Scorpio','Sagittarius']
bmonth = input('plese enter your birth month: ').lower()
bday = int(input('please enter your birth day: '))
def zodiac(m,d):
    if d > 31 or d <= 0 or m not in months:
        print('error!')   
    else:
        index = months.index(bmonth)
        if m == months[6] or m == months[7] or m == months[8] or m == months[9]:
            if d < 23:
                choosen = sign[index]
            else:
                choosen = sign[index + 1]
        if m == months[10] or m == months[11]:
            if d > 21 and index == 11:
                choosen = sign[0]
            elif d > 21 and index != 11:
                choosen = sign[index + 1]
            else:
                choosen = sign[index]
        if m == months[2] or m == months[4] or m == months[5]:
            if d < 21:
                choosen = sign[index]
            else:
                choosen = sign[index + 1]
        if m == months[0] or m == months[3]:
            if d < 20:
                choosen = sign[index]
            else:
                choosen = sign[index + 1]
        if m == months[2]:
            if d < 19:
                choosen = sign[index]
            else:
                choosen = sign[index + 1]
    print('your sign is: ' + choosen)
zodiac(bmonth,bday)

File: projects/m1/013-birth-date-to-astrological-sign/solutions/valentinaBazzacchi/solution-birth-date-to-astrological-sign.js
let month = prompt("Enter your month of birth");

let day = prompt("Enter your day of birth");
month = month.toLowerCase();

if ((month == "december" && day >= 22) || (month == "january" && day <= 19)) {
alert("Your zodiac sign is Capricorn!");

} else if ((month == "january" && day >= 20) || (month == "february" && day <= 18)) {
alert("Your zodiac sign is Aquarius!");

} else if ((month == "february" && day >= 19) || (month == "march" && day <= 20)) {
alert("Your zodiac sign is Pisces!");

} else if ((month == "march" && day >= 21) || (month == "april" && day <= 19)) {
alert("Your zodiac sign is Aries!");

} else if ((month == "april" && day >= 20) || (month == "may" && day <= 20)) {
alert("Your zodiac sign is Taurus!");

} else if ((month == "may" && day >= 21) || (month == "june" && day <= 20)) {
alert("Your zodiac sign is Gemini!");

} else if ((month == "june" && day >= 21) || (month == "july" && day <= 22)) {
alert("Your zodiac sign is Cancer!");

} else if ((month == "july" && day >= 23) || (month == "august" && day <= 22)) {
alert("Your zodiac sign is Leo!");

} else if ((month == "august" && day >= 23) || (month == "september" && day <= 22)) {
alert("Your zodiac sign is Virgo!");

} else if ((month == "september" && day >= 23) || (month == "october" && day <= 22)) {
alert("Your zodiac sign is Libra!");

} else if ((month == "october" && day >= 23) || (month == "november" && day <= 21)) {
alert("Your zodiac sign is Scorpio!");

} else if ((month == "november" && day >= 22) || (month == "december" && day <= 21)) {
alert("Your zodiac sign is Sagittarius!");
}